crates/biome_service/src/configuration/json.rs

```diff
@@ -71,7 +71,7 @@ pub struct JsonFormatter {
 
     /// Print trailing commas wherever possible in multi-line comma-separated syntactic structures. Defaults to "none".
     #[partial(bpaf(long("json-formatter-trailing-commas"), argument("none|all"), optional))]
-    pub trailing_commas: Option<TrailingCommas>,
```

Comment:
I would like to keep this consistent with the rest of the CLI options.

Reply:
I reverted them in c0d3ab2.

There're some failures in CI tests, I'll take a look at them tomorrow.